pkg/sources/github/receive_adapter/receive_adapter.go

```diff
@@ -54,8 +56,27 @@ type GithubSecrets struct {
 // HandlePullRequest is invoked whenever a PullRequest is modified (created, updated, etc.)
 func (h *GithubHandler) HandlePullRequest(payload interface{}, header webhooks.Header) {
 	log.Print("Handling Pull Request")
-	pl := payload.(github.PullRequestPayload)
-	postMessage(h.target, &pl)
+	pl := payload.(gh.PullRequestPayload)
+
+	source := pl.PullRequest.HTMLURL
+
+	var eventType string
+	if len(header["X-GitHub-Event"]) == 1 {
+		switch header["X-GitHub-Event"][0] {
+		case "pull_request":
+			eventType = github.PullrequestEvent
+		default:
+			eventType = "unsupported"
+		}
+
+	}
+
+	eventID := "unknown"
```

Comment:
Also, should this be a UUID or something if not supplied, rather than a deterministic value?

Reply:
added uuid